extensions/arc/deployment/src/test/java/io/quarkus/arc/test/config/ConfigMappingTest.java

```diff
@@ -38,37 +38,39 @@
     @RegisterExtension
     static final QuarkusUnitTest TEST = new QuarkusUnitTest()
             .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
-                    .addAsResource(new StringAsset("config.my.prop=1234\n" +
```

Comment:
Is is possible to revert this (format?) change? It makes it very hard to review :)

Reply:
Yes. Not sure what happened.